README.md

```diff
@@ -1 +1,49 @@
-# ERSU-Bot
\ No newline at end of file
+# ERSU-Bot
+A Telegram Bot for UNICT students aided at helping students find informations about ERSU services.
+
+## How does it work?
+
+##Requirements
```

Comment:
here is needed a space between `##` and the title, same below

use vscode markdown previewer to get the result immediately.

![image](https://user-images.githubusercontent.com/519778/196647235-ae208d4c-9da3-4928-bd52-2e69887a594a.png)